### PR TITLE
Clarify javadoc for AckedClusterStateUpdateTask

### PR DESCRIPTION
**Pull request by jakelandis**
Javadoc only change to clarify "acknowledged"

### CODE REVIEW

File: server/src/main/java/org/elasticsearch/cluster/AckedClusterStateUpdateTask.java [hunk 1]
```diff
@@ -15,8 +15,9 @@
 import org.elasticsearch.core.TimeValue;
 
 /**
- * An extension interface to {@link ClusterStateUpdateTask} that allows to be notified when
- * all the nodes have acknowledged a cluster state update request
+ * An extension interface to {@link ClusterStateUpdateTask} that allows the caller to be notified after the master has
+ * accepted, processed, committed, and published the cluster state update AND after the rest of the nodes
```
Comment (DaveCTurner) [inline server/src/main/java/org/elasticsearch/cluster/AckedClusterStateUpdateTask.java:19]: I would describe it as: process, publish, accept, commit, apply (and then ack). Although "process" is a little vague, maybe "computed" would be better.

File: server/src/main/java/org/elasticsearch/cluster/AckedClusterStateUpdateTask.java [hunk 2]
```diff
@@ -15,8 +15,9 @@
...
+ * (or a specified subset) have also accepted, processed, and acknowledged the cluster state update.
```
Comment (DaveCTurner) [inline server/src/main/java/org/elasticsearch/cluster/AckedClusterStateUpdateTask.java:20]: The other nodes accept and then apply the state before acking it. Again, "process" is a little vague.

**Review by DaveCTurner — COMMENTED** [review]
I left some suggestions

**Comment by jakelandis** [comment]
thanks David ... I updated the wording in https://github.com/elastic/elasticsearch/pull/103001/commits/2c4b479c51f35ff31a825e55ad12cc4b463505c0

**Review by DaveCTurner — APPROVED** [review]
LGTM

**Pull request merged**